gradeforge/utils.py

```diff
@@ -131,12 +131,6 @@ def get_semester_today():
     return parse_semester(get_season_today())
 
 
-def save(obj, output=stdout):
-    '''Put some data on disk'''
-    with open(output, 'wb') as i:
-        i.write(obj)
-
-
 def army_time(time):
```

Comment:
Not related to PR #47 per se, but for time-related functionality, we might want to look into using dateutil for safety and correctness.

Reply:
This is an excellent point I didn't even consider. Adding it as an issue now.